### DISCUSSION TITLE
Auto boot without a disc

### DISCUSSION THREAD

**Discussion by Darklordsidius** [emukidid/swiss-gc#748] [Q&A]
I'm trying to understand how to do this but something in my brain isn't clicking. 

I have the xenogc chip installed & the sd2... adapter. I want to boot Swiss straight from the micro sd card when powered on, without having to use/create a disc but I can't figure it out. I didn't used to be this dense but I had a bad accident with head injuries and now something like this is so freaking difficult. 

I cannot seem to find an exact tutorial on my specific desired boot up style. Can someone please assist?

**Comment by rapperskull** [comment]
It's not your fault, it's just not possible. The only thing the XenoGC chip can do is to boot a burned disc (or a disc from another region). There's no way to load Swiss automatically, unless you burn it to the disc.

**Reply by Darklordsidius** [reply]
Awesome, thank you very much.

**Reply by Extrems** [reply]
The XenoGC is deprecated now that we have PicoBoot.

**Comment by maniaco007** [comment]
Using PicoBoot :)
